### ISSUE TITLE
Simplify the starter template

### ISSUE THREAD

**Issue by dotMorten** [dotnet/maui#26484]
### Description

I'd like to suggest that the starter template stop including "tutorial" content like the dotnet_bot.png*, how a button click works etc. This is all junk code getting thrown in the project that you will be forced to remove every-single-time. I've seen over and over again that while people have deleted the xaml, the [dotnet_bot.png file is still left in place and deployed with apps all over the place](https://github.com/search?q=dotnet_bot.png&type=code).
I also think it doesn't serve much of a purpose trying to tutor people how to do these basic items in the project template - that's what a getting started tutorial is for, and likely what people would start with first, not to mention once you created one project, you're past that and it is yet again just annoyance.

Bottom line is that it creates unnecessary friction when getting started on a new project. You recently introduced the new "Include sample content" checkbox, so let the sample content just be in that version and remove the extra sample stuff from the non-sample version of the template (not to mention the include sample content also deploys the dotnet_bot.png but never actually uses it).

It also creates consistency with [existing WPF template](https://github.com/dotnet/wpf/blob/c6bff274b42ea186c4a53aab6e6447cf92aeb1e1/packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/net6.0/MainWindow.xaml#L9-L11), UWP, as well as [the work that is currently happening in the WinUI project template](https://github.com/microsoft/WindowsAppSDK/pull/4911).

* I have nothing against our beloved dotnet bot. However I hate seeing him getting deployed out to all those apps to live a life of sitting idle never, getting displayed to the world and living a life of broken dreams.

### Public API Changes

None

### Intended Use-Case

Reducing friction when starting a new .NET MAUI project.

**Comment by jfversluis** [comment]
I have to agree [with Jon](https://github.com/dotnet/maui/pull/26448#issuecomment-2528301735) that I'm not sure if we want to rip out everything just yet and immediately. There is now a lot of content out there that shows this template as the starting position when you begin your .NET MAUI journey. I think this whole change makes sense from a more experienced developer standpoint, however from the more beginner standpoint I would think that this has value? But that is based on nothing more than gut feeling. I'd be good to measure that maybe somehow.

As a sort of transition can we start with introducing something like `--empty` parameter? Both in Visual Studio as well as the CLI tooling so you can kind of get a dropdown with: 

* Full sample app (Syncfusion control fuelled one)
* Basic sample (the one that is default now)
* Empty (nothing will be there, we'd introduce this one as part of this change)

That way at least people will have some way of making sure that nothing is included that shouldn't be there. I could see happening that a tire-kicking person would start with either option 0 or 1 and then when they're really going to start building their own app they choose option 2 and we'll be sure that .NET bot isn't included. Can we make that some kind of metric? Scanning binaries to see if its in there and we see the number decrease? 🤔 

But more seriously @davidortinau can we see in telemetry what option people would choose from the above? We could track that for a while and see how people behave with this.

I think by adding the empty option we'd give you a way to have less friction and if we indeed see that people don't really care about the simple sample, we can start taking that out and make the empty the new default.

**Comment by dotMorten** [comment]
> Basic sample (the one that is default now)

I still hold the position that a project template should not be a sample app. That's what samples, doc and tutorials should be for. It's also really weird to not check the "include sample content" checkbox, yet I still get sample content.

Wrt to "full sample app", again this shouldn't be a sample app, but a fully set up typical more advanced app starting point with your view models and typical views, but that is mostly solvable by just not calling it "sample content".

> I think this whole change makes sense from a more experienced developer standpoint, however from the more beginner standpoint I would think that this has value?

I won't argue it doesn't have a little value for the very first run. But already at the second run this is just stuff that is in the way.

If anything I think the very least dotnet-bot needs to go (sorry - I swear still love you dotnet bot!). It's an easy thing to forget to delete, after deleting the xaml that everyone else would delete. It's already in literally 1000s of places on github alone, because people forgot to clean this up, not realizing it was added in the first place. That's not helpful for the beginners.

**Comment by MAUIoxo** [comment]
Reading over this, the proposal from @jfversluis having a drop-down to select from is a good compromise in my opinion. It’s still questionable what the default should be then? Probably from reading over @dotMorten it should then be the most basic as people will get more advanced over time and then probably really don’t want to get rid of things mentioned. But, on the other hand, for some example projects, I still also like having the dotnet_bot in there. It also does not look unprofessional or so.

**Comment by mattleibow** [comment]
Looking at the Aspire and ASP.NET Core templates, they have 2 separate ones for 
 - starter
 - empty

We already have too many templates so I suppose we can keep the single template and maybe have a dropdown to avoid conflicts with someone checking the boxes:

App Content:
* Starter Sample Content
* Basic Content
* Empty
